--- otp/lib/kernel/src/wrap_log_reader.erl.orig
+++ otp/lib/kernel/src/wrap_log_reader.erl
@@ -116,11 +116,11 @@
 close(#wrap_reader{fd = FD}) ->
     file:close(FD).
 
--type chunk_ret() :: {Continuation2, Terms :: [term()]}
-                   | {Continuation2,
+-type chunk_ret() :: {continuation(), Terms :: [term()]}
+                   | {continuation(),
                       Terms :: [term()],
                       Badbytes :: non_neg_integer()}
-                   | {Continuation2, 'eof'}
+                   | {continuation(), 'eof'}
                    | {'error', Reason :: term()}.
 
 -spec chunk(Continuation) -> chunk_ret() when
